SPECS/python-yamlloader/python-yamlloader.spec

```diff
@@ -0,0 +1,90 @@
+%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+%bcond_without  python2
+%define pkgname yamlloader
+
+Summary:        Loaders and dumpers for PyYAML
+Name:           python-%{pkgname}
+Version:        0.5.4
+Release:        1%{?dist}
+License:        MIT
+URL:            https://github.com/Phynix/yamlloader
+Vendor:         Microsoft
+Distribution:   Mariner
+#Source0:       https://github.com/Phynix/%{pkgname}/archive/%{version}.tar.gz
+Source0:        %{pkgname}-%{version}.tar.gz
+BuildArch:      noarch
+
+%description 
+This module provides loaders and dumpers for PyYAML. 
+
+
+%if %{with python2}
```

Comment:
I didn't realize at the time, but this is a Fedora relic that we don't use and don't plan to ever use in 1.0. Any new python packages in this PR should have been made without Python2 support (and if they were, they can be safely removed). Can we remove any instances of these python2 versions (for new packages only) in this PR?

Reply:
Removed all instances in this PR.